vsir.html

```diff
@@ -828,7 +732,35 @@
             const contentId = selectedSong.hash;
             const objectId = contentId.startsWith("iq__") ? contentId : "";
             const versionHash = contentId.startsWith("hq__") ? contentId : "";
-    
+
+
+            if (isFirstLoad) {
+              document.getElementById("embed-player").insertAdjacentHTML('beforeend', `
+                  <iframe id=embed
```

Comment:
this is all we're really doing here. putting an iframe embed that works on iphone, to compare